src/program.js

```diff
@@ -114,6 +115,11 @@ export class Program {
         throw error;
       }
     }
+
+    if (argv.verbose) {
```

Comment:
was this causing an error? It should go back to the top of the `try` block because it needs to happen before any other log statement is called.